controller/work_item_board_blackbox_test.go

```diff
@@ -0,0 +1,62 @@
+package controller_test
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/fabric8-services/fabric8-wit/app/test"
+	. "github.com/fabric8-services/fabric8-wit/controller"
+	"github.com/fabric8-services/fabric8-wit/gormapplication"
+	"github.com/fabric8-services/fabric8-wit/gormtestsupport"
+	"github.com/fabric8-services/fabric8-wit/resource"
+	testsupport "github.com/fabric8-services/fabric8-wit/test"
+	tf "github.com/fabric8-services/fabric8-wit/test/testfixture"
+	"github.com/goadesign/goa"
+	uuid "github.com/satori/go.uuid"
+	"github.com/stretchr/testify/suite"
+)
+
+type workItemBoardSuite struct {
+	gormtestsupport.DBTestSuite
+	svc     *goa.Service
+	ctrl    *WorkItemBoardController
+	testDir string
+}
+
+func TestWorkItemBoardSuite(t *testing.T) {
+	resource.Require(t, resource.Database)
+	suite.Run(t, &workItemBoardSuite{
+		DBTestSuite: gormtestsupport.NewDBTestSuite(),
+	})
+}
+
+// The SetupTest method will be run before every test in the suite.
+func (s *workItemBoardSuite) SetupTest() {
+	s.DBTestSuite.SetupTest()
+	s.svc = testsupport.ServiceAsUser("Board-Service", testsupport.TestIdentity)
+	s.ctrl = NewWorkItemBoardController(s.svc, gormapplication.NewGormDB(s.DB))
```

Comment:
You don't need to use `gormapplication.NewGormDB(s.DB)`. You could just use `s.GormDB`

Reply:
Fixed.